### ISSUE TITLE
Add generation of osm.bz2 file for Overpass API

### ISSUE THREAD

**Issue by PanierAvide** [developmentseed/osm-seed#268]
Hello,

Overpass API docker needs an OSM.BZ2 file ([here](https://github.com/developmentseed/osm-seed/blob/develop/envs/.env.overpass.example#L6)) which, as far as I see, isn't generated in any image of the repository.

I guess this could be done after [this command](https://github.com/developmentseed/osm-seed/blob/develop/images/planet-dump/start.sh#L26) by converting OSM.PBF file into OSM.BZ2, for example with

```
osmosis --read-pbf-fast \
	$local_planetPBFFile \
	--write-xml \
	${local_planetPBFFile/.pbf/.bz2}
```

Or maybe it is intended that user generates this by himself ? Maybe this could be documented in Overpass image readme.

Best regards.

**Comment by Rub21** [comment]
@PanierAvide  Thank you for reporting it, osm-seed Overpass is base in the repo: https://github.com/wiktorn/Overpass-API,  in case you pass the PBF file you can add the env var to pre-process  convert pbf to osm.bz2 file,  e.g

```
OVERPASS_PLANET_PREPROCESS='mv /db/planet.osm.bz2 /db/planet.osm.pbf && osmium cat -o /db/planet.osm.bz2 /db/planet.osm.pbf && rm /db/planet.osm.pbf'
```

I have added same CLI in the env var file in the .env file for overpass: https://github.com/developmentseed/osm-seed/pull/269

**Comment by PanierAvide** [comment]
Great, thanks for updating the env template :+1:

**Comment by batpad** [comment]
Fixed in #269 ! Thank you so much @PanierAvide and @Rub21 ! (Please re-open if I misunderstood the fix)
